package/kedro_viz/models/experiment_tracking.py

```diff
@@ -51,11 +53,7 @@ class TrackingDatasetGroup(str, Enum):
     JSON = "json"
 
 
-# The keys will match a dataset type with any prefix, e.g.
-# plotly.plotly_dataset.PlotlyDataSet would include the following:
-# plotly.plotly_dataset.PlotlyDataSet
-# plotly.plotly_dataset.PlotlyDataSet
-# my.custom.path.to.plotly.plotly_dataset.PlotlyDataSet
+# Map dataset types to their group
```

Comment:
```suggestion
# Map dataset types (as produced by get_dataset_type) to their group
```

Reply:
Added in [4774238](https://github.com/kedro-org/kedro-viz/pull/1232/commits/4774238a726340a700f532bec40747187d20aebe).